cmake/RepositoryDependenciesSetup.cmake

```diff
@@ -65,16 +66,16 @@ ENDIF()
 IF ((NOT ${KDD_INT_LONG}) AND (NOT ${KDD_INT_LONG_LONG}))
   IF ("${${PROJECT_NAME}_ENABLE_STKBalance}" STREQUAL "")
     # STKBalance may be enabled but only implicitly (as a dependence of STK);
-    # give a warning but turn off STKBalance support
-    MESSAGE(WARNING "int global indices are enabled in Trilinos. "
-                    "Because STKBalance requires long or long long "
-                    "global indices, STKBalance will be disabled.  "
-                    "To make this warning go away, do not request "
-                    "int global indices in Trilinos (that is,  do not "
-                    "set Tpetra_INST_INT_INT=ON or "
-                    "Tpetra_INST_INT_UNSIGNED=ON)." )
+    # give a message but turn off STKBalance support
+    MESSAGE("NOTE:  int global indices are enabled in Trilinos. "
```

Comment:
I could go either way but I think "NOTE:" is a better choice in this case so thanks for making that change.